azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -23,39 +26,40 @@ def __init__(self, pipeline):
 
         :param pipeline: The pipeline that the client will use.
         """
+        # We pass the pipeline through instead of the auth provider to allow
+        # for future support of different protocols - the pipeline will come in
+        # already configured.
         self._pipeline = pipeline
```

Comment:
>self._pipeline = pipeline [](start = 8, length = 25)

OK, this answers a question I had on Oliva's review.  Still feels awkward, but I get it.

Reply:
Actually no. Next PR will move away from this due to the need to support multiple pipelines for multiple destinations.

---
In reply to: [294460475](https://github.com/Azure/azure-iot-sdk-python-preview/pull/110#discussion_r294460475) [](ancestors = 294460475)